flowetl/tests/integration/conftest.py

```diff
@@ -0,0 +1,168 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# -*- coding: utf-8 -*-
+"""
+Conftest for flowetl integration tests
+"""
+import os
+import shutil
+
+from time import sleep
+from subprocess import DEVNULL, Popen
+from pendulum import now, Interval
+from airflow.models import DagRun
+
+import pytest
+import docker
+
+
+@pytest.fixture(scope="session")
+def docker_client():
+    """
+    docker client object
+    """
+    return docker.from_env()
+
+
+@pytest.fixture(scope="session")
+def flowetl_tag():
+    """
+    Get flowetl tag to use
+    """
+    return os.environ.get("FLOWETL_TAG", "latest")
+
+
+# pylint: disable=redefined-outer-name
```

Comment:
Why is this pylint directive needed? Where is this other place where `flowetl_container` is defined?

Reply:
It's not it's the fixtures...above....

Reply:
```
tests/integration/conftest.py:38:22: W0621: Redefining name 'docker_client' from outer scope (line 22) (redefined-outer-name)
tests/integration/conftest.py:38:37: W0621: Redefining name 'flowetl_tag' from outer scope (line 30) (redefined-outer-name)
```